actionmailbox/lib/action_mailbox/engine.rb

```diff
@@ -29,5 +30,22 @@ class Engine < Rails::Engine
         ActionMailbox.ingress = app.config.action_mailbox.ingress
       end
     end
+
+    initializer "action_mailbox.actionable_errors" do
+      ActiveSupport::ActionableError.define :InstallError, under: ActionMailbox do |actionable|
```

Comment:
Still think these should be `:MissingInstallError` since there's not been an error when installing it.